rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/SonarLintLauncherBuilder.java

```diff
@@ -62,7 +62,8 @@ public GsonBuilder getDefaultGsonBuilder() {
           .registerTypeHierarchyAdapter(Instant.class, new InstantTypeAdapter())
           .registerTypeHierarchyAdapter(UUID.class, new UuidTypeAdapter())
           .registerTypeHierarchyAdapter(URI.class, new UriTypeAdapter())
-          .registerTypeHierarchyAdapter(Duration.class, new DurationTypeAdapter());
+          .registerTypeHierarchyAdapter(Duration.class, new DurationTypeAdapter())
+          .serializeNulls();
```

Comment:
I think I understand Kirill's point. If the key is not in the map, it means we have no name for a given project key, I don't see how having a key with a null value provides more details. I checked the VSCode [implementation](https://github.com/SonarSource/sonarlint-vscode/blob/master/src/connected/connections.ts#L164) and it should work even with "missing" keys in the map. But no big deal.

Reply:
My point of view is that if the client sends a list of keys, we send them the same amount of keys back. We expressed that we did some operations for all the keys and didn't forget anything.
I will merge it with the `serializeNulls()`, but I don't hold a strong opinion on that. @damien-urruty-sonarsource & @kirill-knize-sonarsource, feel free to revert it if you don't see the above as valid.